package.js

```diff
@@ -694,44 +694,59 @@ class PathToTarkovController {
-                            locations[i].base.exits[x].RequirementTip = '';
-                        }
-                        if (locations[i].base.exits[x].RequiredSlot) {
-                            delete locations[i].base.exits[x].RequiredSlot;
-                        }
-                    }
-
-                    // Make all extractions available to extract
-                    if (locations[i].base.exits[x].Name !== "EXFIL_Train") {
+                    if (locations[i].base.exits[x].Name !== "EXFIL_Train" && !locations[i].base.exits[x].Name.includes("lab") || locations[i].base.exits[x].Name === "lab_Vent") {
+                        // Make all extractions available to extract
                         if (locations[i].base.exits[x].Chance !== 100) {
                             locations[i].base.exits[x].Chance = 100;
                         }
+                        
+                        // Remove extracts restrictions
+                        if (this.config.remove_all_exfils_restrictions) {
+                            if (locations[i].base.exits[x].ExfiltrationTime !== 10) {
+                                locations[i].base.exits[x].ExfiltrationTime = 10;
+                            }
+                            
+                            if (locations[i].base.exits[x].PlayersCount !== 0) {
+                                locations[i].base.exits[x].PlayersCount = 0;
+                            }
+                            
+                            if (locations[i].base.exits[x].Id !== '') {
+                                locations[i].base.exits[x].Id = '';
+                            }
+                            
+                            if (locations[i].base.exits[x].Count !== 0) {
+                                locations[i].base.exits[x].Count = 0;
+                            }
+                            
+                            if (locations[i].base.exits[x].PassageRequirement !== "None") {
+                                locations[i].base.exits[x].PassageRequirement = "None";
+                            }
+                            
+                            if (locations[i].base.exits[x].ExfiltrationType !== "Individual") {
+                                locations[i].base.exits[x].ExfiltrationType = "Individual";
+                            }
+                            
+                            if (locations[i].base.exits[x].RequirementTip !== '') {
+                                locations[i].base.exits[x].RequirementTip = '';
+                            }
+                            
+                            if (locations[i].base.exits[x].RequiredSlot) {
+                               delete locations[i].base.exits[x].RequiredSlot;
+                            }
+                        }
                     }
                 }
             }
         }
+        
+        if (!this.config.remove_all_exfils_restrictions) {       

```

Comment:
This line break the default config requirements.

When I ran the game with the default config file, exfils are corrects.

Here, we want to set our customs exits points when this option is true.

Sorry the code is not very comprehensible here and don't reflect what `remove_all_exfils_restrictions` really means.

The correct line of code is:
```js
if (this.config.remove_all_exfils_restrictions) {
```

Reply:
sounds good I was running test now so we just need to flip that check?

Reply:
Exactly ! see [my comment here](https://github.com/guillaumearm/PathToTarkov/pull/2#issuecomment-1063213817) for more details